packages/core/src/tools/create-request-client.js

```diff
@@ -24,6 +24,7 @@ const createRequestClient = (befores, afters, options) => {
   const httpAfters = [];
 
   if (!options.skipDefaultMiddle) {
+    httpBefores.push(sanitizeHeaders);
```

Comment:
I don't think it is necessary here any longer. You are already pushing it in the `befores` arg in both places the function `createRequestClient` is called. Moreover, the flow will never reach here for both callers because `skipDefaultMiddle` is set to `true` for both.